docs/sources/shared/hugo-error-example-bad-link.md

```diff
@@ -0,0 +1,29 @@
+headless: true
+labels:
+  products:
+    - oss
```

Comment:
@jdbaldry Is prettier suggestion correct?

Reply:
I think prettier is confused because this page is missing the opening front matter delimiter (`---`) at the start of the file.

Reply:
Ah! Fixed that, but it's still saying it needs a description even though it's a headless file. Is that right? I can add one just to appease the validator.

Reply:
Yeah unfortunately the validator doesn't know that shared files are unique and that the description is useless there.

I guess it might be nice to reuse the description as part of any shared prompt in the future. The page "TITLE" with description "DESCRIPTION" is reused in the following places: ...